apache2/msc_logging.c

```diff
@@ -1174,6 +1174,9 @@ void sec_audit_logger_json(modsec_rec *msr) {
             yajl_kv_bool(g, "response_body_dechunked", 1);
         }
 
+#ifdef LOG_NO_PRODUCER
+        if (msr->txcfg->debuglog_level < 9) return;
```

Comment:
Please check this condition here. It does not seems to be doing what you aiming to do in the patch/commit description.